nix/release/docker-hub-full-description.md

```diff
@@ -15,13 +15,8 @@ write from scratch.
 To learn how to use this container, see the [PostgREST Docker
 documentation](https://postgrest.com/en/stable/install.html#docker).
 
-You can configure the PostgREST image by setting the enviroment variables used
-in the default `/etc/postgrest.conf` file or overriding that file. This is the
-default configuration file:
-
-```
-$DEFAULT_CONFIG
-```
+You can configure the PostgREST image by setting
+[enviroment variables](https://postgrest.org/en/stable/configuration.html).
```

Comment:
This link (and the changed docker hub description) will only be correct/useful, when we have released the next stable version. Once we release the next nightly, this will be out of sync. We should maybe skip the `postgrest-release-dockerhubdescription` step for nightly releases to make sure the description always stays in sync with the stable release?

@steve-chavez @monacoremo WDYT?

Reply:
Yes, we should skip postgrest-release-dockerhubdescription for the nightly builds